cmd/lakefs/cmd/diagnostics.go

```diff
@@ -2,69 +2,16 @@ package cmd
 
 import (
 	"log"
-	"os"
 
 	"github.com/spf13/cobra"
-	"github.com/treeverse/lakefs/pkg/block/factory"
-	"github.com/treeverse/lakefs/pkg/catalog"
-	"github.com/treeverse/lakefs/pkg/diagnostics"
-	"github.com/treeverse/lakefs/pkg/kv"
-	"github.com/treeverse/lakefs/pkg/stats"
-	"github.com/treeverse/lakefs/pkg/upload"
 )
 
-// diagnosticsCmd represents the diagnostics command
 var diagnosticsCmd = &cobra.Command{
```

Comment:
Why not totally remove this command?

Reply:
Thought we would like to use it as some point - but I agree that it is strange to keep something without any functionality.